setup.cfg

```diff
@@ -77,8 +77,6 @@ doctest_plus = enabled
 doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
 # TODO we probably need to get rid of the following...
 filterwarnings =
-    once::DeprecationWarning
-    once::PendingDeprecationWarning
```

Comment:
I wholeheartedly support this change, now that I think about it.  ...as well as after doing an `import this`.

> Errors should never pass silently.
> Unless explicitly silenced.

Reply:
Yeah, I'll try to tackle the removal of the other warning filters next. Come to think about it, we're probably exposing any potential users to a whole lot of Coupling or RelativityWarnings we have no idea about.